lmfit/minimizer.py

```diff
@@ -1622,22 +1634,7 @@ def leastsq(self, params=None, max_nfev=None, **kws):
         from the covariance matrix.
 
         This method calls :scipydoc:`optimize.leastsq` and, by default,
-        numerical derivatives are used, and the following arguments are
```

Comment:
I think it is OK to remove this table of parameters from the docstring, but....

I do sort of feel like maybe we should have better documentation on how to tweak tolerances, 'epsfcn' step sizes, and number of function evals. I think that is sort of a separate topic promoted by the removal of this part of the docstring. I think we might need more "user guide" documentation.

Reply:
Yes, it would be worthwhile to add an example to that gallery to show how to pass things to the optimizers using `fit_kws`. Not sure if I'll have time/interest to do that immediately and since it's would be part of the documentation that can be updated in between releases anyway so shouldn't hold up releasing a new version. 

I'll merge this PR and try to go through the other methods to possibly update keywords and take care of new ones in later SciPy versions if present.